### ISSUE TITLE
Custom character set for hint labels config error

### ISSUE THREAD

**Issue by asmir-abdulahovic** [aidanholm/luakit#312]
When I try to change $HOME/.config/luakit/rc.lua to:

-- Use a custom charater set for hint labels
local s = follow.label_styles
follow.label_maker = s.sort(s.reverse(s.charset("asdfqwerzxcv")))

I get:

[    0.012767] I: luaH_loadrc:163: Loading rc: /home/akill/.config/luakit/rc.lua
[    0.096648] I: /usr/share/luakit/lib/adblock.lua:90: adblock: Found 0 rules lists
[    0.107550] E: luaH_dofunction:147: Lua error: attempt to index global 'follow' (a nil value)
                  Traceback:
                  (1) /home/akill/.config/luakit/rc.lua:115 in main chunk
[    0.107578] W: luaH_parserc:229: loading rc '/home/akill/.config/luakit/rc.lua' failed, falling back...

**Comment by aidanholm** [comment]
It's currently not possible to change that the hint label generator from the config file. This is because label generation is done in the web process, not the main UI process, and I haven't yet added the ability to send Lua functions between the two processes (#314).

You can edit the default label maker directly; it's defined in `lib/select_wm.lua`. You may need to copy `select_wm.lua` to `~/.config/luakit/` before you can edit it. Note that doing so might break in the future if I modify `lib/select_wm.lua` and your setup continues to use an old version.

I will probably add IPC function serialization (#314) support soon. I'm considering eventually switching to using Cairo to render the hint overlay, in the web process, which will fix #54 and #56.

**Comment by asmir-abdulahovic** [comment]
I see, I'll edit select_wm.lua and update it for now.
Thank you.

**Comment by aidanholm** [comment]
@Asm1r I've merged #315, so you should be able to add the following to `config/rc.lua` now, without changing `select_wm.lua`:
```
local select = require "select"
select.label_maker = function (s)
    return s.sort(s.reverse(s.charset("asdfqwerzxcv")))
end
```

**Comment by asmir-abdulahovic** [comment]
That is great. Though I've found another problem which I'll describe in an other issue.
